chain/rust/src/plutus/utils.rs

```diff
@@ -489,6 +495,34 @@ pub fn compute_total_ex_units(redeemers: &[LegacyRedeemer]) -> Result<ExUnits, A
     Ok(sum)
 }
 
+impl TryFrom<u64> for Language {
+    type Error = DeserializeError;
+
+    fn try_from(language: u64) -> Result<Self, Self::Error> {
```

Comment:
I am also thinking of just getting rid of the `Language` enum entirely and swapping it with just `pub type Language = u64` then maybe a `Languages` module with V1/V2/V3 members to avoid having to convert to/from `u64`/`Language` via `From`/`TryFrom`